### DISCUSSION TITLE
Updating Firebase built-in libraries

### DISCUSSION THREAD

**Discussion by PrakharDoneria** [Sketchware-Pro/Sketchware-Pro#993] [Q&A]
The Libraries which comes built-in are too old and thus does not support latest android releases. Can you tell me a procedure to update my libraries with the latest firebase release.

**Comment by j-px** [comment]
You'll need a Sketchware Pro version newer than v6.4.0-rc05 for this, which currently means you should get the latest nightly build.

Download the specific library you want to upgrade with Local library Manager, then exclude the version that's automatically added by Sketchware Pro to your project (otherwise there will be conflicts) as described in #696.

**Comment by PrakharDoneria** [comment]
Does this mean I need to put the command
>
> Message ID: <Sketchware-Pro/Sketchware-Pro/repo-discussions/993/comments
> ***@***.***>
>
com.google.firebase:firebase-admin:9.2.0

As per the documentation of firebase listed in

https://firebase.google.com/docs/admin/setup

If not please provide the command
